### PR TITLE
Fixed Bookmark Dark Mode and Added Support for Bookmark Translation

### PR DESCRIPTION
**Pull request by Thunder-Blaze**
### Description
- Added Id Key pair for Bookmark Translations
- Fixed Bookmarks Dark Mode

### CODE REVIEW

File: languages.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ const translations = {
         "enable_ai_tools": "Show shortcuts for AI tools",
         "googleAppsMenuText": "Google Apps",
         "googleAppsMenuInfo": "Show shortcuts for Google Apps",
+        // Todo List
```
Comment (prem-k-r) [inline languages.js:32]: can you please move todoplaceholder and hover here too

File: languages.js [hunk 2]
```diff
@@ -83,6 +84,13 @@ const translations = {
         "location": "Earth",
         // End of Weather
 
+        // Bookmarks
```
Comment (prem-k-r) [inline languages.js:87]: Please do for bookmark delete confirm alert too

File: style.css [hunk 3]
```diff
@@ -638,7 +638,7 @@ body {
 	align-items: center;
 	padding: 8px 32px;
 	& h3 {
-		font-size: 1.2rem;
+		font-size: 1rem;
```
Comment (prem-k-r) [inline style.css:641]: can you please add a dot in css line 14 `./`

**Review by prem-k-r — APPROVED** [review]

**Review by itz-rj-here — APPROVED** [review]
Perfect

**Comment by itz-rj-here** [comment]
@Thunder-Blaze do the changes that @prem-k-r requested. It’s needed actually.

File: languages.js (hunk 1 above)
Comment (Thunder-Blaze) [reply languages.js:32]: sure

File: style.css (hunk 3 above)
Comment (Thunder-Blaze) [reply style.css:641]: done

**Comment by prem-k-r** [comment]
Also arrow is not responsive.. lets make it position fixed?

**Comment by itz-rj-here** [comment]
> Also arrow is not responsive.. lets make it position fixed?

It’s just an idea from my own. Maybe ask @XengShi for this.

**Comment by prem-k-r** [comment]
lets decrease size?
image

```
style="transform: scale(0.88); transform-origin: center;"
```

File: languages.js (hunk 2 above)
Comment (Thunder-Blaze) [reply languages.js:89]: done

**Comment by Thunder-Blaze** [comment]
@prem-k-r @itz-rj-here I used scale property rather than transform as it would've overwritten the styles in style.css which make the arrow flip

**Comment by prem-k-r** [comment]
@itz-rj-here please review and merge both pr

**Comment by itz-rj-here** [comment]
> @itz-rj-here please review and merge both pr

I can't i can only able to confirm the codes. 🙂
Still lemme see.

**Review by itz-rj-here — APPROVED** [review]

**Pull request merged**